idea/tests/org/jetbrains/kotlin/idea/quickfix/CommonIntentionActionsTest.kt

```diff
@@ -168,6 +168,20 @@ class CommonIntentionActionsTest : LightPlatformCodeInsightFixtureTestCase() {
         """.trim().trimMargin(), true)
     }
 
+    fun testRemoveConstructorParameters() {
+        myFixture.configureByText("foo.kt", """
+        |class <caret>Foo(i: Int) {
+        |}
+        """.trim().trimMargin())
+
+        myFixture.launchAction(codeModifications.createAddCallableMemberActions(MethodInsertionInfo.constructorInfo(
+                atCaret<UClass>(myFixture), makeParams())).findWithText("Remove 1st parameter from method 'Foo'"))
```

Comment:
Ok. So could I now cherry-pick it to `1.1.4_172`-branch? Or what are the relationships between `master_172` and `1.1.4_172` now?